WeakAuras/Prototypes.lua

```diff
@@ -7897,6 +7897,228 @@ Private.event_prototypes = {
     automaticrequired = true
   },
 
+  ["Spell in Range"] = {
+    type = "status",
+    events = {
+      ["events"] = {
+        "PLAYER_TARGET_CHANGED",
```

Comment:
There are events missing for other types of unit changes.

Reply:
I think Infus refers to 
https://github.com/WeakAuras/WeakAuras2/blob/72c8bb70b69920010525121ace197e5572517aac/WeakAuras/Prototypes.lua#L2057-L2064

Reply:
minus AddUnitRoleChangeInternalEvents line